test/token/ERC20/ERC20.behavior.js

```diff
@@ -40,7 +40,7 @@ function shouldBehaveLikeERC20 (errorPrefix, initialSupply, initialHolder, recip
       describe('when the recipient is not the zero address', function () {
         const to = anotherAccount;
 
-        describe('when the spender has enough approved balance', function () {
+        describe('when the spender has enough approved allowance', function () {
```

Comment:
"approved allowance" doesn't sound right to me. What about "when the spender has enough allowance"?